md/BonitaBPM_platform_setup.md

```diff
@@ -82,50 +91,26 @@ Remember that in the [Tomcat](tomcat-bundle.md) and [WildFly](wildfly-bundle.md)
 
 To do so, proceed as follows:
 
-1. Create a new empty database.
-2. Configure the *Platform setup tool* as explained above.
-3. From the tool, run `setup.sh init` (Unix/Mac) or `setup.bat init` (Windows).
+   1. Create a new empty database.
+   2. Configure the *Platform setup tool* as explained above.
+   3. From the tool, run `setup.sh init` (Unix/Mac) or `setup.bat init` (Windows).
 
 This last step will connect to the database pointed to in `database.properties`, create the tables required for the platform to run, push all configuration from the `platform_conf/initial` folder and push the licenses from the licenses folder.
 
 That's it! You can now run Bonita BPM Platform on this database.
 
 
-<a id="update_platform_conf" />
-
-## Update Bonita BPM Platform configuration or licenses
-
-If you want to modify the configuration of an already initialized Bonita BPM Platform, you must use the *Platform setup tool* as follows:
-
-1. Stop Bonita BPM Platform.
-2. If not already done, update the `database.properties` of the *Platform setup tool* to point to the database used by Bonita BPM Platform.
-3. Run `setup.sh pull` or `setup.bat pull` from the tool. It will get the current configuration and put it in the `platform_conf/current` folder.
-4. Modify the configuration files inside the `platform_conf/current` folder according to your needs.
-5. If you are updating your license file, put it inside `platform_conf/licenses`, along with the existing ones. If some retrieved license files are not valid anymore, you can remove them, to delete them from the database when pushed.
-6. Run `setup.sh push` or `setup.bat push` from the tool.
-7. Start Bonita BPM Platform.
-
-
-::: warning
-Note that the *Platform Setup tool* does not need the Bonita BPM Server to be running for the configuration to be updated. However, the server needs to be restarted for the changes to take effect.
-:::
-
-::: warning
-Keep in mind that the folder **`platform_conf/initial`** is not used anymore once the platform has been initialized for the first time. To update your configuration, only the **`platform_conf/current`** folder is taken into account.
-:::
-
-
 <a id="run_bundle_configure" />
 
-## Run the Bonita BPM bundle configuration (Tomcat 7 / WildFly 10)
+## Configure the server (Tomcat 7 / WildFly 10)
 
 ::: info
 In the [Tomcat](tomcat-bundle.md) and [WildFly](wildfly-bundle.md) bundles, configuration is done automatically when calling `start-bonita.sh / .bat`
 :::
 
 1. Update the file `database.properties` with the database properties matching your environment (db vendor name, server name, server port, connection username and password)
```

Comment:
Why not say **Configure the Platform setup tool as explained above.** like for `init` command?